components/consentRequestForm/styles.js

```diff
@@ -182,5 +201,48 @@ export default function styles(theme) {
       justifyContent: "flex-start",
       padding: "1rem",
     },
+
+    "date-container": {
+      display: "flex",
+      padding: theme.spacing(0.7, 1.2),
+      border: `1px solid ${theme.palette.grey.A100}`,
+      borderRadius: 10,
+      height: "2.5rem",
+      marginBottom: "1rem",
+      marginTop: "0.5rem",
+      position: "relative",
+      maxWidth: 300,
+    },
+
+    "date-input": {
+      fontSize: "1rem",
+      width: "100%",
+      font: "inherit",
+      fontWeight: 500,
+      color: theme.palette.primary.text,
+    },
+
+    "date-button": {
+      padding: 0,
+    },
+
+    "date-picker": {
+      maxWidth: 320,
+      borderRadius: 4,
+      boxShadow: `0 4px 8px 0 ${theme.palette.grey[500]}`,
+      overflow: "hidden",
+      display: "flex",
+      flexDirection: "column",
+      position: "absolute",
+      left: 0,
+      top: "calc(100% + 0.5rem)",
+      zIndex: 2,
+      backgroundColor: "white",
```

Comment:
minor thing, but we have white in the theme

Reply:
👀  will update

Reply:
@VirginiaBalseiro the theme does not seem to have `white` configured in the palette. Checking in prism even there `color: white` is used instead. Can we keep it as is in this PR and raise a ticket to change that? It would require updating both prism and PB usages as there are multiple in both repos 😬

Reply:
It's actually `theme.palette.background.paper`

Reply:
It's a minor thing but helpful if we ever want to change the theme. We should not be using hard coded colors for this reason (unless they're not meant to change with the theme, but background and font colors usually do).